### PR TITLE
added codeowners file

### PR DESCRIPTION
**Pull request by craig-osterhout**
### Proposed changes

Adding /.github/CODEOWNERS file to automatically notify the specified users when pull requests are created for specified folders/files.

### CODE REVIEW

**Review by aevesdocker — APPROVED** [review]
LGTM - thank you Craig!

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+/compose/ @dockertopia @usha-mandya
+
+/compose/install.md @aevesdocker
+
+/desktop/ @aevesdocker @usha-mandya
+
+/docker-hub/ @jerae-duffin
+
+/docker-hub/orgs.md @craig-osterhout
```
Comment (usha-mandya) [inline .github/CODEOWNERS:14]:
```suggestion
/docker-hub/orgs.md @craig-osterhout @jerae-duffin 
```

**Comment by usha-mandya** [comment]
Thanks @craig-osterhout. Looks good overall. Added some minor suggestions

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Each line is a file pattern followed by one or more owners.
+# Owners will be requested for review when someone opens a pull request.
+
+# For more details, see https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+/compose/ @dockertopia @usha-mandya
```
Comment (dockertopia) [inline .github/CODEOWNERS:6]:
```suggestion
/compose/ @dockertopia @usha-mandya
/engine/ @dockertopia @usha-mandya
```

**Review by dockertopia — APPROVED** [review]
Thanks @craig-osterhout!

File: .github/CODEOWNERS (hunk 1 above)
Comment (craig-osterhout) [reply .github/CODEOWNERS]: Removed this entry. With the more general /docker-hub/ entry, we'll both be notified if this file changes.

**Pull request merged**